tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -624,6 +654,28 @@ stages:
           condition: succeededOrFailed()
           displayName: 
 
+      - task: CmdLine@2
+        inputs:
+          script: |
+            docker run \
+              --user 0 \
```

Comment:
>    --user 0 \ [](http://example.com/codeflow?start=10&length=14)

why run as root?

Reply:
It was suspected that u0 is needed to pip install onnxruntime. It turned to be unneeded